Loop/Managers/LoopDataManager.swift

```diff
@@ -1355,7 +1355,7 @@ extension LoopDataManager {
 
         let tempBasal = predictedGlucose.recommendedTempBasal(
             to: glucoseTargetRange,
-            at: now(),
+            at: predictedGlucose[0].startDate,
```

Comment:
Hm... this looks like a significant change to me.  Is there an explanation I can read?  A unit test?

Reply:
Here is the original PR for this fix: https://github.com/LoopKit/Loop/pull/1235. There is also some discussion in the linked issue.

The change aligns LoopDataManager more with what was being tested as well. The tests for recommendedTempBasal almost exclusively use the date of the first predicted glucose (which is current bg).